lib/zendesk_apps_tools/theme.rb

```diff
@@ -88,10 +88,11 @@ def start_listener(callbacks_after_upload)
 
       def generate_payload
         payload = {}
-        templates = Dir.glob(theme_package_path('templates', '*.hbs'))
+        templates = Dir.glob(theme_package_path('templates', '*.hbs')) + Dir.glob(theme_package_path('templates', '*', '*.hbs'))
```

Comment:
I was just wondering, why not use an explicit `/` instead of passing multiple arguments to `theme_package_path`? In line 94 you are matching against `templates/` which includes the forward slash…

Reply:
true, changed